docs/network/edges.html

```diff
@@ -894,7 +898,31 @@ <h3>Options</h3>
             <td>selfReferenceSize</td>
             <td>Number</td>
             <td><code>false</code></td>
-            <td>When the to and from nodes are the same, a circle is drawn. This is the radius of that circle.</td>
+            <td>When the to and from nodes are the same, a circle is drawn. This is the radius of that circle. 
+                This property is deprecated please use selfReference instead.
+            </td>
+        </tr>
+        <tr class='toggle collapsible' onclick="toggleTable('optionTable','selfReference', this);">
+            <td><span parent="arrows" class="right-caret"></span> selfReference</td>
+            <td>Object</td>
+            <td><code>Object</code></td>
+            <td>When the to and from nodes are the same, a circle is drawn. This provides radius of circle and position.
+            </td>
+        </tr>
+        <tr parent="selfReference" class="hidden">
+            <td class="indent">size</td>
+            <td>Number</td>
+            <td><code>false</code></td>
+            <td>This is the radius of circle.
+            </td>
+        </tr>
+        <tr parent="selfReference" class="hidden">
+            <td class="indent">angle</td>
+            <td>Number</td>
+            <td><code>undefined</code></td>
+            <td> This is the angle (Provided in radians) to indicate position for the circle. If position is not provided then it will 
+                be automatically determined.
+            </td>
```

Comment:
Could you add this to the Korean docs, please?

PS: If you don't speak Korean then instead of translating it simply replace the English text with `TODO`. Just so that we know there's something in need of translating. Thanks.